zcncore/wallet.go

```diff
@@ -445,11 +446,12 @@ func GetNetworkJSON() string {
 
 // CreateWallet creates the wallet for to configure signature scheme.
 // It also registers the wallet again to block chain.
-func CreateWallet(statusCb WalletCallback) error {
+func CreateWallet(wg *sync.WaitGroup, statusCb WalletCallback) error {
```

Comment:
```suggestion
func CreateWallet(statusCb WalletCallback) error {
	if len(_config.chain.Miners) < 1 || len(_config.chain.Sharders) < 1 {
		statusCb.OnWalletCreateComplete(StatusError, "", "SDK not initialized")
		return errors.New("", "SDK not initialized")
```

it should works

Reply:
i am just following this pattern so that it follows golang code pattern
This also makes sure that we handle the wg as per expectations (especially when multiple people edit the same code.

Reply:
call wg.Done multiple times, it panics there see https://github.com/0chain/gosdk/runs/5414092105?check_suite_focus=true